library/std/src/rt.rs

```diff
@@ -16,6 +16,7 @@
 
 // Re-export some of our utilities which are expected by other crates.
 pub use crate::panicking::{begin_panic, begin_panic_fmt, panic_count};
+pub use core::panicking::panic_display as begin_panic_display;
```

Comment:
Why import with a rename? That is confusing and makes the code harder to follow.

Reply:
Haha it's really just because all these other functions are named this way. I can remove the rename.